front/src/App.js

```diff
@@ -1,8 +1,26 @@
 import React from "react";
-import "./App.css";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Container from "@material-ui/core/Container";
+
+import { mainRoutes } from "assets/routes";
 
 function App() {
-  return <div>Hello World!</div>;
+  return (
+    <Router>
+      <Container maxWidth="lg">
+        <Switch>
+          {mainRoutes.map((route, index) => (
+            <Route
+              key={index}
```

Comment:
dobra praktyka jest unikac indexow jako kluczy. Jak usuniesz jakis element z tablicy, to przerenderuje sie niepotrzebnie cala lista od tego momentu. W Routach daje osobiscie key={route.path}

Reply:
Done.